devices.js

```diff
@@ -834,7 +834,39 @@ const devices = [
             return {'bottom_left': 1, 'bottom_right': 2, 'top_left': 3, 'top_right': 4};
         },
     },
+    // eCozy
+    {
+        zigbeeModel: ['Thermostat'],
+        model: 'ecozy Smart Thermostat',
+        vendor: 'ecozy',
+        description: 'Smart Thermostat',
+        supports: '',
```

Comment:
Could you fill in this?

Reply:
Are the keyword for "supports" standardized somewhere?